share/spice/skyscanner_flight_search/skyscanner_flight_search.css

```diff
@@ -0,0 +1,38 @@
+/*
+    Document   : zeroclickinfo-spice/share/spice/skyscanner_flight_search/skyscanner_flight_search.css
+    Created on : 2016-09-23 11:50 AM
+    Author     : sheyman
+    Description:
+    Purpose of the stylesheet follows.
+    To change this template use Tools | Templates.
+*/
+
+.tile_title {
```

Comment:
All of these CSS rules need to be namespaced with `.zci-skyscanner_flight_search` -- we create a `div` on the page with a class that matches the ID of the IA for this purpose.

Otherwise this CSS will affect other IAs